### DISCUSSION TITLE
Deposit

### DISCUSSION THREAD

**Discussion by MansorBhoy** [moneymanagerex/moneymanagerex#4175] [General]
Deposit is the first transaction for the day..

**Comment by tactilis** [comment]
@MansorBhoy 
Are you having a problem with MMEX?

> Deposit is the first transaction for the day..

You will need to explain clearly what the issue is.

**Comment by MansorBhoy** [comment]
Hi,
I am using the Windows Desktop version of MMEX and loving it.
My issue is this:
I opened a "My Bank Account" in MMEX. I entered transactions for the day, into account, not taking consideration if it is a withdrawal or deposit. Thus the transactions appears as which entry I key-in first. The result of which, will show as if the account is in negative balance half way the day. A typical example is attached.
It would be nicer if MMEX could automatically assigned all deposits to be the first transaction for the day, so that the above will not be occur.
thanks
    On Saturday, 5 February 2022, 04:54:45 pm MYT, Martin ***@***.***> wrote:  
 
 


@MansorBhoy
Are you having a problem with MMEX?


Deposit is the first transaction for the day..


You will need to explain clearly what the issue is.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by TomAff** [comment]
If I understand you correct...
What about using splitted transaction? One for cash in, one for cash out?

Do you have a screenshot how do you want a have it, and how it is now?

**Comment by tactilis** [comment]
@MansorBhoy 
> I entered transactions for the day, into account, **not taking consideration if it is a withdrawal or deposit.** 
> Thus the transactions appears as which entry I key-in first.
> It would be nicer if MMEX could automatically assigned all deposits to be the first transaction for the day, so that the above will not be occur.

By default, MMEX sorts transactions by date, and then by ID when there are multiple transactions on one day. This will correspond to the order in which the transactions are entered for a particular day.

> The result of which, will show as if the account is in negative balance half way [through] the day.

Why does this matter? The end of day balance will be correct.

If you are really concerned about a negative balance pat way through the day, then _you_ should enter all of your deposits for the day first, then your withdrawals.

> A typical example is attached.

Nothing was attached. To attach screenshots, you need to past them directly into the github issue here https://github.com/moneymanagerex/moneymanagerex/discussions/4175

**Comment by TomAff** [comment]
If you post the income and expenses according to the bank statement, the balance in MMEX will be exactly the same as that in the bank.
Even if it's negative...
Another advantage of this balance calculation is that you can reconcile the balances after each posting.
